### DISCUSSION TITLE
2023 年最受歡迎的十個 Emoji

### DISCUSSION THREAD

**Discussion by alvin119** [1101-GS4719-C/Python-Community-GS4719-C#1831] [閒聊八卦]
身為Z世代，表情符號、gif、貼圖早已離不開我們的生活，
文章中有列出了2023 年最受歡迎的十個 Emoji，
雖然文章中並未寫出是如何統計的，不過我覺得的確排行榜中至少除了第3 7 8個以外自己都很常用，
還是有一定的參考價值。
真的有人真的覺得大拇指那個表情符號有「粗魯」跟「有敵意」的意思嗎?
資料來源 : https://ccc.technews.tw/2024/01/03/2023-top10-emoji/

**Comment by Patsy17** [comment]
我最喜歡用愛心眼睛跟融化的符號誒，都沒有上榜xd
平常用哭笑的符號都是跟比較不熟的人才會使用

**Comment by yuchien-lin** [comment]
除了在榜上的第一個以外其他在榜上的我都沒有用過，此外我比較常用的表情還有🤯這個表情符號跟🥲，所以這個榜看起來可能跟我不太符合。

**Comment by sorn32** [comment]
我是1跟2還有大拇指都很常用，我覺得大拇指很好用阿，雖然我同學會覺得我在敷衍_(:3 」∠ )_
但有時候不知道要說啥，就用大拇指就好，真的很方便

**Reply by FakeWood** [reply]
"有時候不知道要說啥，就用大拇指就好"
這感覺就是在敷衍沒錯啊哈哈XD

**Comment by RavenJiaSin** [comment]
我一般來說和別人用訊息聊天時，會模仿對方的聊天方式和習慣。所以在表情符號的使用上會看對方常用什麼，這樣比較不會踩雷，但有一說一，😂跟🤣真的超級好用，不知道要回啥時就🤣🤣🤣🤣🤣

**Comment by lyykey** [comment]
個人最喜歡無辜的表情 🥺🥺，當今天跟對方聊著聊著，覺得對方突然愛回不回的時候，就可以傳這個給他。
如果要要詢問對方要不要跟你出門或去吃飯，別人拒絕時，也可以發一下🥺🥺。
總而言之，有種情緒勒索的感覺。

**Comment by SusuTima** [comment]
我最喜歡用的表符🙏😊😄😆😭😢😿🥹，有時候有表情符號的幫助會讓對話看起來不那麼嚴肅，所以認真覺得表情符號是個好用的東東。

**Comment by tshsu04224** [comment]
我最常用的十個大概是😓😅🥲😆🥹🥺😻😤🫠🤔，怎麼感覺好像跟大家都不太一樣，雖然榜上的幾個也是會用啦，但好像就沒有這麼頻繁，我覺得我個人奇怪的地方在於太常用的就沒那麼想用了，像是😻就是因為😍太常見，所以想用點不一樣的代替。一句話能有很多種不同的情緒，感覺表情符號的存在，就是為了讓傳訊息時的對方能理解你現在的語氣，才不會讓訊息失去感情。

**Comment by YANRONG0123** [comment]
我最常用的大概是🤣或是😭還有😴，但這其實也蠻看心情的，至於用法是什麼，我認為是因人而異，以及跟不同人對話時，所代表的意義也會有些微的不同，面對不同意思，我只能表示尊重。

**Comment by Eileen050824** [comment]
我自己看了榜單後，是覺得跟我使用習慣蠻合的(我常用的是1、2、3、5、9)，特別是哭笑不得的emoji是真的和朋友聊天的時候很好用！我覺得emoji最大的好處除了避免尷尬，也不會像文字一樣那麼冰冷直接，也比較不會被人家誤會。不過，話說我和身邊朋友都算常用比讚emoji，這是少數跟這個數據不太一樣的。

**Comment by YifenLin** [comment]
其實排行榜上第四名以後的，以我的經驗都很少在聊天室用到ㄟ。另外，我常常跟朋友聊天的時候會用顏文字。例如，(^▽^)表示開心或興奮，(￣︶￣)↗表示自信或得意等等。

**Comment by William08290829** [comment]
表情符號在聊天中扮演了重要的角色。它們能夠補充文字無法表達的情感和語氣，使對話更加生動有趣。此外，表情符號還能夠幫助溝通者更好地理解對方的意思，減少誤解和衝突的可能性。同時，它們通常只需要很少的空間，卻能夠傳達豐富的信息，這有助於節省時間和提高效率。

**Comment by cassy0201** [comment]
表情符號真的對打字簡短的人來說是一大福音啊

**Comment by buoioiry** [comment]
哇哇常用的完全都不在前十名，不過裡面稍微有在用的是🔥，可以表達很多意思包含生氣發火跟這個東西超讚，差不多是這種意思，平常還喜歡用的表情符號也有🫶🏻，比一般的愛心可愛而且對不熟的人用也不會太尷尬，超實用！
